tools/amd_build/build_caffe2_amd.py

```diff
@@ -14,6 +14,8 @@
     "caffe2/transforms/*",
     "caffe2/video/*",
     "caffe2/distributed/*",
+    "caffe2/queue/*",
```

Comment:
Thanks for the catch! I am assuming all the binary tests are covered here. let me know if that is not your intention.

Reply:
Yes the pattern `caffe2/**/*_test*` covers all the c++ tests. This will help us automatically opt-in to any new c++ tests for rocm.